### PR TITLE
Bring Quickhelp up to date

### PR DESCRIPTION
**Pull request by hjerning**
The Quickhelp was very useful to me since I had forgotten a lot of key combinations and the Alt + K dialogue is very verbose. It was however quite out of date, so the main focus with this PR is to remove invalid keys and update existing entries, not to document all new features.

### CODE REVIEW

**Comment by mywave82** [comment]
I will try to get these double-checked during the weekend, and also reminds to test the filters.

**Review by mywave82 — COMMENTED** [review]
It still adjust "intensity" or scale as the "ALT+K" current wording is, but the current scale is not displayed in the header of the analyzer.

So we need to adjust the text in the help file and add current scale into the analyzer heading inside the player

**Comment by mywave82** [comment]
(The filters I wanted to check out are correct)

Great work going through these tedious details by the way!

* Please, add yourself to the CREDITS file
* Need to add the correct text to ctrl+page up/down for analyzer
* Adding scale to the analyzer heading I can add unless you add it ( cpiface/cpianal.c )
* Maybe also adjust the range of the scale for the analyzer if possible - the current range seems subtle for the end-user ?

**Comment by hjerning** [comment]
> It still adjust "intensity" or scale as the "ALT+K" current wording is, but the current scale is not displayed in the header of the analyzer.
> 
> So we need to adjust the text in the help file and add current scale into the analyzer heading inside the player

The wording "intensity" is from the v1.666 changelog and scale is probably more suitable. I noticed that Ctrl + PgUp/PgDown still worked from the analyser to change the scale, but as you say there is no presentation of the scale in the UI, so I though of it as a hidden gem that didn't belong in the Quickhelp and removed the old keyboard combination, just as for [pattern view](https://github.com/mywave82/opencubicplayer/pull/98/commits/a23b3cebc2cb36deec9ae1d0cf593788eb7b4b92), instead of updating the description.

But I will update commit f59e51e46380684bb8acaa486191afadcdcfb2d9 and replace the description `scroll instruments (for multiview)` with `change scale` instead of removing the line then. The commit message could actually be exactly the same, but it's probably better to rephrase it to more clearly state that the "intensity" adjustments are still used today.

**Comment by mywave82** [comment]
After spending the night, I think the best word to use is "gain"

Scale in music software can be referred to "musical scale", octave, ...

**Comment by hjerning** [comment]
A very good point, but gain could also be confused with amplification (which I just noticed is wrongly spelled as amplication in the UI). What about `change dB scale`?

**Comment by hjerning** [comment]
> * Please, add yourself to the CREDITS file
> * Need to add the correct text to ctrl+page up/down for analyzer
> * Adding scale to the analyzer heading I can add unless you add it ( cpiface/cpianal.c )
> * Maybe also adjust the range of the scale for the analyzer if possible - the current range seems subtle for the end-user ?

The first two points are now done, the last two I leave for you to decide on how to implement.

**Comment by mywave82** [comment]
Thank you for your patience and effort
